seqr/views/apis/individual_api_tests.py

```diff
@@ -58,7 +72,49 @@ def test_edit_individuals(self):
                             set(response_json['familiesByGuid']['F000001_1']['individualGuids']))
         self.assertListEqual(response_json['familiesByGuid']['F000003_3']['individualGuids'], [])
 
-        self.assertSetEqual({ID_UPDATE_GUID, FAMILY_UPDATE_GUID, CHILD_UPDATE_GUID}, set(response_json['individualsByGuid']))
+        self.assertSetEqual({ID_UPDATE_GUID, FAMILY_UPDATE_GUID, CHILD_UPDATE_GUID},
+                            set(response_json['individualsByGuid']))
         self.assertEqual(response_json['individualsByGuid'][ID_UPDATE_GUID]['individualId'], UPDATED_ID)
         self.assertEqual(response_json['individualsByGuid'][ID_UPDATE_GUID]['maternalId'], UPDATED_MATERNAL_ID)
         self.assertEqual(response_json['individualsByGuid'][CHILD_UPDATE_GUID]['paternalId'], UPDATED_ID)
+
+    def test_delete_individuals(self):
+        individuals_url = reverse(delete_individuals_handler, args=[PROJECT_GUID])
+        _check_login(self, individuals_url)
+
+        # send invalid requests
+        response = self.client.post(individuals_url, content_type='application/json', data=json.dumps({
+            'individualsX': [INDIVIDUAL_IDS_UPDATE_DATA]
+        }))
+        self.assertEqual(response.status_code, 400)
+
+        # send valid requests
+        response = self.client.post(individuals_url, content_type='application/json', data=json.dumps({
+            'individuals': [INDIVIDUAL_IDS_UPDATE_DATA]
+        }))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['individualsByGuid', 'familiesByGuid'])
+
```

Comment:
whenever you test delete, you need to test that deleting non-existent individuals fails

Reply:
Deleting a non-existent individual is always successful.